tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -211,7 +211,7 @@ def test_find_pod_labels(self):
             do_xcom_push=False,
         )
         self.run_pod(k)
-        self.client_mock.return_value.list_namespaced_pod.assert_called_once()
+        self.client_mock.return_value.list_namespaced_pod.assert_called()
```

Comment:
Not sure if this provides much value any longer? Should we check the call count?

Reply:
Should we assert that it's called twice?
```suggestion
        assert self.client_mock.return_value.list_namespaced_pod.call_count == 2
```

Reply:
Sounds good 👍